modules/framework/galasa-parent/dev.galasa.framework.api.users/src/test/java/dev/galasa/framework/api/users/internal/routes/UsersDeleteRouteTest.java

```diff
@@ -0,0 +1,180 @@
+package dev.galasa.framework.api.users.internal.routes;
+
+import java.time.Instant;
+import java.util.Map;
+
+import javax.servlet.ServletOutputStream;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.Test;
+
+import dev.galasa.framework.api.common.BaseServletTest;
+import dev.galasa.framework.api.common.EnvironmentVariables;
+import dev.galasa.framework.api.common.HttpMethod;
+import dev.galasa.framework.api.common.InternalUser;
+import dev.galasa.framework.api.common.mocks.MockAuthStoreService;
+import dev.galasa.framework.api.common.mocks.MockEnvironment;
+import dev.galasa.framework.api.common.mocks.MockFramework;
+import dev.galasa.framework.api.common.mocks.MockFrontEndClient;
+import dev.galasa.framework.api.common.mocks.MockHttpServletRequest;
+import dev.galasa.framework.api.common.mocks.MockHttpServletResponse;
+import dev.galasa.framework.api.common.mocks.MockTimeService;
+import dev.galasa.framework.api.common.mocks.MockUser;
+import dev.galasa.framework.api.users.mocks.MockUsersServlet;
+import dev.galasa.framework.spi.auth.IInternalUser;
+import dev.galasa.framework.spi.utils.GalasaGson;
+
+public class UsersDeleteRouteTest extends BaseServletTest {
+
+
+    GalasaGson gson = new GalasaGson(); 
```

Comment:
A GalasaGson instance should be available in the BaseServletTest class so this can be removed